packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -37,6 +38,12 @@ const ApiTokenCreateView = () => {
   const { lockApp, unlockApp } = useOverlayBlocker();
   const toggleNotification = useNotification();
   const history = useHistory();
+  const { trackUsage } = useTracking();
+  const trackUsageRef = useRef(trackUsage);
+
+  useEffect(() => {
+    trackUsageRef.current('didEditTokenFromList');
+  }, []);
```

Comment:
after testing it, I think it calls trackUsage with didEditTokenFromList even if the user is creating.

maybe changing it this way would fix it:
`  useEffect(() => {
     if(!isCreating) {
       trackUsageRef.current('didEditTokenFromList');
     }
  }, []);
`

or maybe we could event move the logic directly into the UpdateButton like this?
`
    <LinkUpdate
      to='PATH'
      title='TITLE'
      onClick={() => trackUsageRef('didEditTokenFromList')}
    >
      <Pencil />
    </LinkUpdate>
`

Reply:
Based on the list of events the `didEditTokenFromList` is supposed to be triggered:
> When the user lands on the create or edit token page 

But I we could definitely add two new events:
- `willCreateTokenFromList`
- `didCreateTokenFromList`

That way we have different events for the creation and update action done from the ListView. 

WDYT?

Ping @JabStrapi.

Reply:
Oh right I didn't know that, thanks for the explanation!
Indeed having willCreateTokenFromList and didCreateTokenFromList as separate events could be interesting :)

Reply:
Let's add two new events yes! But I'd recommend to go for `willAddTokenFromList` and `didAddTokenFromList`.